core/src/main/java/ai/timefold/solver/core/impl/domain/variable/listener/support/VariableListenerSupport.java

```diff
@@ -111,6 +127,7 @@ private void processShadowVariableDescriptorWithListVariable(ShadowVariableDescr
 
     private void
             processShadowVariableDescriptorWithoutListVariable(ShadowVariableDescriptor<Solution_> shadowVariableDescriptor) {
+        // TODO: Set up variable listener for declarative shadow variables
```

Comment:
Should this still be here? I'd expect this feature to be in a mergable state, and therefore without TODOs. Possible future improvements will be tracked separately.